src/main/java/org/cloudfoundry/promregator/lite/discovery/SimpleTargetResolver.kt

```diff
@@ -0,0 +1,191 @@
+package org.cloudfoundry.promregator.lite.discovery
+
+import org.cloudfoundry.promregator.lite.cfaccessor.SimpleCFAccessorImpl
+import kotlinx.coroutines.async
+import kotlinx.coroutines.awaitAll
+import kotlinx.coroutines.coroutineScope
+import org.cloudfoundry.client.v2.routes.Route
+import org.cloudfoundry.promregator.lite.config.CloudFoundryConfig
+import org.cloudfoundry.promregator.lite.config.PromregatorConfiguration.Companion.DEFAULT_ID
+import org.springframework.stereotype.Component
+import java.util.*
+import java.util.concurrent.atomic.AtomicBoolean
+import org.cloudfoundry.promregator.lite.config.Target
+import kotlin.text.RegexOption.IGNORE_CASE
+
+
+data class OrgResponse(
```

Comment:
Note these data classes replace all the previous intermediate targets and other temp classes used when using the reactive approach. They are all immutable (val) properties so we can build up the results from the CF queries easily.